ceph_medic/collector.py

```diff
@@ -122,6 +122,14 @@ def collect():
     failed_nodes = 0
     for node_type, nodes in cluster_nodes.items():
         for node in nodes:
+            # check if a node type exists for this node before doing any work:
+            try:
+                metadata[node_type]
+            except KeyError:
+                msg = "Skipping host group: {}".format(node_type)
```

Comment:
`Skipping node {} from unknown host group: {}.format(node, node_type)` 

Sorry I should've noticed this earlier. Otherwise we will see this same line for 10 different nodes and can't tell them apart